### ISSUE TITLE
Prompt input window breaks coc-codeaction-selected

### ISSUE THREAD

**Issue by fannheyward** [neoclide/coc.nvim#2582]
## Reproduce the bug

0. mapping

```
xmap <leader>a  <Plug>(coc-codeaction-selected)
nmap <leader>a  <Plug>(coc-codeaction-selected)
```

1. use `coc-react-refactor`
1. `v` to select a code block
1. `<leader>a` to fire code action
1. prompt user to input component name
1. code selection is missing, `workspace.nvim.call('visualmode')` returns empty

**Comment by chemzqm** [comment]
Maybe there is a way to fix this, you can pass `document` and `Range` as `arguments` for your Command to fix this https://github.com/fannheyward/coc-react-refactor/blob/master/src/index.ts#L7

**Comment by chemzqm** [comment]
Fixed by fed2bf12
